src/test/java/com/epam/ld/module2/testing/MessengerTest.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.ld.module2.testing;
+
+import com.epam.ld.module2.testing.template.Template;
+import com.epam.ld.module2.testing.template.TemplateEngine;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Spy;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+public class MessengerTest {
+
+    @Spy
```

Comment:
Using annotation and then initializing it with spy(...) method call is confusing